src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -91,7 +91,8 @@ public enum TargetFramework
         Net50,
         Net60,
         Net70,
-        Net80
+        Net80,
+        Net90,
```

Comment:
Consider adding a test or upgrading an existing one to exercise this new TFM

Reply:
All the places for the new tests are in the ref generic interfaces branch. For merge purposes it's better to have the initial commit here. Once Aleksey merges this in we'll move to this TFM. If there are any issues I will deal with the fallout.